modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMDataDefinitionConverterImpl.java

```diff
@@ -94,6 +102,25 @@ public String convertDDMFormDataDefinition(
 		return DDMFormSerializeUtil.serialize(ddmForm, _ddmFormSerializer);
 	}
 
+	@Override
+	public String convertDDMFormDataDefinition(
+			String dataDefinition, long groupId, long parentStructureId,
+			long parentStructureLayoutId, long structureId)
+		throws Exception {
+
+		DDMForm ddmForm = DDMFormDeserializeUtil.deserialize(
```

Comment:
Can we call the deserialize directly in the convertDDMFormDataDefinition call?

Reply:
This class already has _ddmFormDeserializer as its reference. If I deserialize directly in the call, I would have to repeat the same reference in the other class. Can we let it this way?